dashboard/src/index.css

```
@@ -4,6 +4,19 @@
 
 @custom-variant dark (&:is(.dark *));
 
+@layer utilities {
```

```suggestion
@layer components {
```
this is not an utility